ui/src/containers/AppDetailsContainer.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import AppsTable from '../components/AppsTable';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { Checkbox } from '@patternfly/react-core';
+import { getApps, reverseAppDetailsTableSort } from '../actions/actions-ui';
+
+export class AppDetailsContainer extends React.Component {
+  constructor (props) {
+    super(props);
+    this.onSort = this.onSort.bind(this);
+    this.onRowClick = this.onRowClick.bind(this);
+    this.getTable = this.getTable.bind(this);
+  }
+
+  componentWillMount () {
+    //this.props.getApps();
+  }
+
+  handleChange = (e) => {};
+
+  onRowClick = (event, rowId, props) => {
```

Comment:
If we use a separate class you can remove the `OnRowClick` event altogether as we wont want to offer that in the detailed app table.

Reply:
@laurafitzgerald can you elaborate on what you mean by a seperate class?

Reply:
Yes, I see what you mean now. If we opt for an `AppDetailsContainer` we can remove it from `AppDetailsContainer`

Reply:
removed now so resolving